backend/src/api/session/session.model.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
```

Comment:
We don't need session model. We've already discussed, that we're saving tokens in user model